app/lib/primer/forms/action_menu.html.erb

```diff
@@ -1,6 +1,6 @@
 <%= render(FormControl.new(input: @input)) do %>
   <%= render(Primer::Alpha::ActionMenu.new(**@input.input_arguments)) do |menu| %>
-    <% menu.with_show_button { "Select..." } %>
+    <% menu.with_show_button("aria-describedby": @label_id) { "Select..." } %>
```

Comment:
I wonder if it would be more semantically correct to update the `aria-labelledby` or `aria-label` so the full accessible name that's announced is something like `Favorite city: Seattle`... but I could also see that could have unintended consequences. 🤔 
What do you think?